boards/arduino-due/include/periph_conf.h

```diff
@@ -342,6 +343,34 @@ extern "C" {
 #define GPIO_B21_MAP        31
 /** @} */
 
+/**
+ * @name PWM configuration
+ * @{
+ */
+#define MCK_DIV_LB_MAX      (10U)
```

Comment:
It's the maximum binary logarithm for the divisor of the prescaled clocks (MCK/2^n -> MCK/1...MCK/1024). Hence the "LB" in the name.
And it seems reasonable to put it in "pwm.c".
There were several macros where I was unsure whether to place them in "pwm.c" or "periph_conf.h". I'm still not got the context of "periph_conf.h" completely.
Is it just board specific -desptite that the cpu ist on board also-, or belongs to peripheral devices on the microcontroller too?